morceus-data/latin/stems/verbs/vbs.latin

```diff
@@ -17037,7 +17061,12 @@
 :le:transmuto
 :de:trans-mu_t	are_vb
 :le:transnato
-:de:trans-na^t	are_vb
+:vs:trans-na^t	conj1
+:vs:trans-na_v   perfstem
```

Comment:
I do see some `transnatav-` perfect stems (https://latin.packhum.org/search?q=transnatav) 
let me know what you think but I think the `trans-na_v perfstem` should belong under the lemma `transno` instead.

Reply:
That was just me not paying attention. I've fixed it and also added the trans- variant to trano for good measure.